client/my-sites/checkout/src/components/bundled-domain-notice.jsx

```diff
@@ -55,7 +55,18 @@ function getCopyForBillingTerm( cart ) {
 	);
 }
 
-export default function BundledDomainNotice( { cart } ) {
+/**
+ * Use showBundledDomainNotice to manage BundleDomainNotice visibility when called.
+ * @param {*} cart
```

Comment:
I assume you had to put this here to satisfy TypeScript. There's a slightly more accurate way to use types in plain JS,

```suggestion
 * @param {import('@automattic/shopping-cart').ResponseCart} cart
```